src/index.js

```diff
@@ -2,48 +2,25 @@
 import _ from 'lodash';
 /* eslint-enable */
```

Comment:
- [ ] Please, consider removing all linter disables. Let the linter does it's direct job - find bugs in the early steps of your project 👍

Reply:
@elmar8287, I used the "eslint-disable" for the Lodash only. besides linters can't pass if I don't disable it 😕

Reply:
@ifeanyi305 in this case you can keep this disabled and go on. So, I see you have got approved the PR. Good luck 👍